docs/recipes/register_cluster.adoc

```diff
@@ -0,0 +1,81 @@
+== Register clusters
+
+Before CBShell can be used to perform data operations on a cluster, the cluster must be registered with the shell.
+Cluster registration can be done through the https://couchbase.sh/docs/#_the_config_dotfiles[config file], or using https://couchbase.sh/docs/#_cb_env_register[cb-env register].
+This recipe shows how to register all the healthy clusters in the active https://couchbase.sh/docs/#_the_config_dotfiles[registered organization].
+
+Firstly we need to get a list of all the healthy clusters, along with the projects to which they belong:
```

Comment:
en_US would be "First, " rather than "Firstly.

Reply:
Chagned